azure-pipelines.sds.pr-release.yml

```diff
@@ -29,12 +29,19 @@ stages:
           cluster: $(cluster)
       - group: vh-github-app-credentials
       - name: prTagName
-        value: 'pr-$(System.PullRequest.PullRequestNumber)'
+        value: 'pr-$(System.PullRequest.PullRequestNumber)-$(Build.SourceVersion)'
```

Comment:
You can add the timestamp here as
pr-$(System.PullRequest.PullRequestNumber)-$(Build.SourceVersion)-${{ variables.timestamp }}
I think